docs/source/faq.rst

```diff
@@ -0,0 +1,22 @@
+**************************
+Frequently Asked Questions
+**************************
+
+.. contents:: The following is a list of common questions asked about pyOpenMS.
+
+How can I wrap a new method with pyOpenMS?
+###########################################
```

Comment:
Let's follow existing formatting style guide for pyopenms-docs `.rst`:
<img width="834" alt="Screenshot 2022-05-24 at 4 09 30 PM" src="https://user-images.githubusercontent.com/4143778/170013738-b76b8c2d-39e5-4da6-95eb-4ea4f86a7d53.png">